lib/modules.nix

```
@@ -151,7 +151,7 @@ rec {
 
           _module.freeformType = mkOption {
             # Disallow merging for now, but could be implemented nicely with a `types.optionType`
```

```suggestion
            # Disallow merging for now, but could be implemented nicely with a `types.optionType`
```

Isn't this implemented now?